testdata/jsonschema/basic_object/GenerateAST/ir.json

```diff
@@ -136,6 +136,63 @@
               },
               "Required": false
             },
+            {
+              "Name": "objectAdditionalPropertiesAny",
+              "Type": {
+                "Kind": "scalar",
+                "Nullable": false,
+                "Scalar": {
+                  "ScalarKind": "any"
+                }
+              },
+              "Required": false
+            },
+            {
+              "Name": "objectAdditionalPropertiesTyped",
+              "Type": {
+                "Kind": "map",
+                "Nullable": false,
+                "Map": {
+                  "IndexType": {
+                    "Kind": "scalar",
+                    "Nullable": false,
+                    "Scalar": {
+                      "ScalarKind": "string"
+                    }
+                  },
+                  "ValueType": {
+                    "Kind": "scalar",
+                    "Nullable": false,
+                    "Scalar": {
+                      "ScalarKind": "string"
+                    }
+                  }
+                }
+              },
+              "Required": false
+            },
+            {
+              "Name": "objectEmptyPropertiesIsAny",
```

Comment:
That seems to be the same as `objectNoPropertiesIsAny` but it should in fact be an empty object, with no properties right. This seems to allow everything now